### PR TITLE
unmask sys-fs/genfstab

### PR DESCRIPTION
**Pull request by peeweep**
1. Forward #29758 change to stable.
2. Unmask it.
3. And add myself as a proxy maintainer.

### CODE REVIEW

File: sys-fs/genfstab/metadata.xml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>peeweep@0x0.ee</email>
+		<name>jinqiang zhang</name>
+	</maintainer>
 	<maintainer type="person" proxied="yes">
```
Comment (thesamesam) [inline sys-fs/genfstab/metadata.xml:8]: Please drop scardracs.

Comment (peeweep) [reply sys-fs/genfstab/metadata.xml]: Dropped

**Comment by peeweep** [comment]
Dropped scardracs and rebase to master to fix merge conflict.

**Comment by peeweep** [comment]
Thanks.

**Pull request closed without merging**